singa_auto/model/dev.py

```diff
@@ -192,7 +194,7 @@ def tune_model(
 
 
 def make_predictions(queries: List[Any], task: str,
-                     py_model_class: Type[BaseModel], proposal: Proposal,
+                     py_model_class: Type[BaseModel], proposal: Proposal, fine_tune_dataset_path,
```

Comment:
if other  tasks do not have the fine_tune_dataset_path, will the job fail?

Reply:
This 'fine_tune_dataset_path' is optional, and won't cause other models' failure.